net/uradvd/Makefile

```diff
@@ -0,0 +1,48 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=uradvd
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL=https://github.com/freifunk-gluon/uradvd.git
+PKG_SOURCE_DATE:=2021-08-14
+PKG_SOURCE_VERSION:=9b0da60e27c67305d251b10163e388191d566d7a
+PKG_MIRROR_HASH:=70dac4c156cd98baea92c9f1fb14692737b6c55c2aa0442c272ec9abe8fe06e4
+
+PKG_MAINTAINER:=Moritz Warning <moritzwarning@web.de>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/uradvd
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=Advertise an IPv6 prefix/route
+endef
+
+define Package/uradvd/description
+	Advertise an IPv6 prefix from the node
```

Comment:
This description is pretty useless, and also too specific ("node" doesn't make any sense in a generic OpenWrt system).